style_links.json

```diff
@@ -124,6 +124,11 @@
     "name": "Yoga Template 2",
     "link": "Yoga Template 2/index.html"
   },
+
+  {
+    "name": "Food Template New",
+    "link": "Food_Template/index.html"
+  },
   
```

Comment:
add it on appropriate place (alphabetically)

Reply:
sure, i'll have to remove the vscode file and add it alphabetically  ,anything else?

Reply:
nope

Reply:
done ,know you can give it a look!